src/charm.py

```diff
@@ -59,14 +67,18 @@ def _configure_cilium(self):
             return
 
         log.info("Applying Cilium manifests")
+        self._configure_cilium_cni()
+        self._configure_hubble()
```

Comment:
`configure_cilium_cni` and `configure_hubble` both seem to swallow a lot of exceptions that would indicate they aren't installed.  

How are they retried

Reply:
If anyone fails to install, the `update-status` event will retry to install both. Nevertheless, I think if Hubble is enable and failed to deploy, the `update-status` event logic will not retry. I'll update the logic to address this case.

Do you think there's a better way to handle this behavior?

Reply:
well, i went with a prodigious use of `event.defer()`

any callback that results in a `WaitingStatus` kinda needs an `event.defer()` to retry if necessary

see uses of `_ops_wait_for` in https://github.com/charmed-kubernetes/charm-kube-virt/blob/main/src/charm.py#L102

Reply:
Cool, I'll rewrite the retry logic to use `event.defer()`. Thanks @addyess 😄